by-language/scala-spark-http/README.md

```diff
@@ -0,0 +1,24 @@
+# Scala example loading data from Spark to CrateDB via http
+
+This example aims to demonstrate how a Spark data frame can be loaded into CrateDB using the CrateDB http endpoint.
+
+It assumes there is a CrateDB instance running on localhost accepting connections with the default `crate` superuser, and it relies on the following table being created:
+
+.. code-block:: sql
+
+    CREATE TABLE myschema.mytable (
+        examplefield1   TIMESTAMP,
+        anotherfield    geo_point,
+        jsonpayload     OBJECT
+    );
+
+In environments with very high volumes of data you may want to consider replacing scalaj.http with async calls with `akka.http` or `AsyncHttpClient`.
+You may also want to explore if connection pooling is useful in your environment, and if JDBC calls leveraging the PostgreSQL wire protocol are more convenient for your particular case.
+
+You can run this example with [sbt]:
+
+.. code-block:: shell
+
+    sbt run
```

Comment:
### About Testing

Unfortunately, [Pueblo's universal test runner](https://github.com/pyveci/pueblo/tree/v0.0.9/pueblo/ngr) does not understand Scala-based projects yet.

We may be able to whip it into the GHA workflow recipe instead, but that is not a blocker for merging this patch.